setup.py

```diff
@@ -83,6 +83,5 @@
         "Topic :: Software Development :: Libraries"
     ],
     long_description=long_description,
-    ext_modules=ext_modules,
-    include_package_data=True
```

Comment:
is ``include_package_data=True`` removed on purpose?

Reply:
I remember I got some trouble with it at the beginning but now it's resolved. Let me put it back real quick!